prbt_hardware_support/test/include/prbt_hardware_support/pilz_manipulator_mock.h

```diff
@@ -33,39 +33,33 @@
  */
 class ManipulatorMock
 {
-  public:
-
-    void advertiseHoldService(ros::NodeHandle nh, std::string hold_service_name);
-    void advertiseUnholdService(ros::NodeHandle nh, std::string unhold_service_name);
-    void advertiseHaltService(ros::NodeHandle nh, std::string halt_service_name);
-    void advertiseRecoverService(ros::NodeHandle nh, std::string recover_service_name);
-
-    void shutdownHoldService();
-    void shutdownUnholdService();
-
-    void advertiseServices(ros::NodeHandle nh,
-                           std::string hold_service_name,
-                           std::string unhold_service_name,
-                           std::string halt_service_name,
-                           std::string recover_service_name);
-
-
-    MOCK_METHOD2(holdCb,        bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
-    MOCK_METHOD2(unholdCb,      bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
-    MOCK_METHOD2(haltCb,        bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
-    MOCK_METHOD2(recoverCb,     bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
-
-  private:
-    bool holdCb_internal(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp);
-    bool unholdCb_internal(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp);
-    bool haltCb_internal(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp);
-    bool recoverCb_internal(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp);
-
-    ros::ServiceServer hold_srv_;
-    ros::ServiceServer unhold_srv_;
-    ros::ServiceServer halt_srv_;
-    ros::ServiceServer recover_srv_;
-
+public:
+  void advertiseHoldService(ros::NodeHandle nh, std::string hold_service_name);
+  void advertiseUnholdService(ros::NodeHandle nh, std::string unhold_service_name);
+  void advertiseHaltService(ros::NodeHandle nh, std::string halt_service_name);
+  void advertiseRecoverService(ros::NodeHandle nh, std::string recover_service_name);
+
+  void shutdownHoldService();
+  void shutdownUnholdService();
+
+  void advertiseServices(ros::NodeHandle nh, std::string hold_service_name, std::string unhold_service_name,
+                         std::string halt_service_name, std::string recover_service_name);
+
+  MOCK_METHOD2(holdCb, bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
+  MOCK_METHOD2(unholdCb, bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
+  MOCK_METHOD2(haltCb, bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
+  MOCK_METHOD2(recoverCb, bool(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp));
+
+private:
+  bool holdCb_internal(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& resp);
```

Comment:
Is it possible to keep the formatting of such structures, for readability reasons?

Reply:
yes, with the comments ..
```
// clang-format off
...
// clang-format on
```